python/dlisio/ext/core.cpp

```diff
@@ -418,10 +418,11 @@ PYBIND11_MODULE(core, m) {
             s.read( static_cast< char* >( info.ptr ), off, n );
             return b;
         })
-        .def( "extract", [](dl::stream& s, const std::vector< long long >& tells) {
+        .def( "extract", [](dl::stream& s,
+                            const std::vector< long long >& tells_indices) {
             std::vector< dl::record > recs;
-            recs.reserve( tells.size() );
-            for (auto i : tells) {
+            recs.reserve( tells_indices.size() );
+            for (auto i : tells_indices) {
```

Comment:
I'm not sure this is actually a clarification. If it's indices anyways, why mix tells into it?

Reply:
I agree, it actually is just "indices" to extract, which would be more clear. But for some reason it has signature of "std::vector< long long >" (which corresponds to tells one) and I am unable to understand where we got it from. 
If that's not a problem, I would happily change it just to "indices".